File: cloud-storage-spring-java/src/main/java/com/github/sevtech/cloud/storage/spring/bean/GetFileResponse.java
package com.github.sevtech.cloud.storage.spring.bean;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Getter;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@SuperBuilder
@JsonInclude(JsonInclude.Include.NON_EMPTY)
public class GetFileResponse extends BaseResponse {
	private byte[] content;
}

File: cloud-storage-spring-kotlin/src/main/kotlin/com/github/sevtech/cloud/storage/spring/bean/GetFileResponse.kt
package com.github.sevtech.cloud.storage.spring.bean

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_EMPTY)
data class GetFileResponse(
        var content: ByteArray? = null,
        var status: Int = 0,
        var cause: String? = null,
        var exception: Exception? = null
) : BaseResponse(status, cause, exception)